### PR TITLE
Remove single quotes from values. Fixes #8

### LINKED ISSUE #8: Single quote in PR title breaks the get-pr-info variables
### Scenario
Given PR title:
> ASDF-111 don't use single quotes in PR title

When using step `ghpr/get-pr-info` and accessing `$GITHUB_PR_BASE_BRANCH` variable
Then variable is an empty line and error message is printed:
```
/tmp/.bash_env-5ff85b683890bc490b71ca94-0-build: line 4: unexpected EOF while looking for matching `''
/tmp/.bash_env-5ff85b683890bc490b71ca94-0-build: line 5: syntax error: unexpected end of file
```

### Details

I've noticed that exporting variables in `get-pr-info` command is not escaping the contents:
```
PR_TITLE=$(echo $PR_RESPONSE | jq -e '.title' | tr -d '"')
echo "PR_TITLE: $PR_TITLE"
echo "export GITHUB_PR_TITLE='$PR_TITLE'" >> $BASH_ENV
```
In my case this resulted in a corrupted env file:
```
echo $BASH_ENV
/tmp/.bash_env-5ff86be9b2b77a7bed582e27-0-build
cat $BASH_ENV
export GITHUB_PR_NUMBER=1568
export GITHUB_PR_TITLE='ASDF-111 don't use single quotes in PR title'
export GITHUB_PR_BASE_BRANCH='develop'
export GITHUB_PR_AUTHOR_USERNAME='jakub'
```
I think single quotes should be escaped to avoid such issues.

### PR DESCRIPTION
**Pull request by jakub-skierbiszewski-signavio**
Remove single quotes from values returned from GitHub. Prevents issues from #8

### CODE REVIEW

**Review by cogat — APPROVED** [review]
This got us today; the change looks clean and it would be great to get it merged.

File: src/commands/get-pr-info.yml [hunk 1]
```diff
@@ -45,15 +45,15 @@ steps:
 
         PR_TITLE=$(echo $PR_RESPONSE | jq -e '.title' | tr -d '"')
```
Comment (jerbob) [inline src/commands/get-pr-info.yml:46]: It seems double quotes are already being trimmed here - maybe it's worth just updating these to trim off single quotes too?
```suggestion
        PR_TITLE=$(echo $PR_RESPONSE | jq -e '.title' | tr -d "\"'")
```

**Review by ns-bdesimone — APPROVED** [review]
LGTM! Thank you for doing this.

File: src/commands/get-pr-info.yml (hunk 1 above)
Comment (CelestineKao) [reply src/commands/get-pr-info.yml:46]: +1 on this! I think it'll be cleaner to have all the string cleaning in one place :)

**Pull request merged**